src/index.html

```diff
@@ -22,6 +30,367 @@
+          </div>
+          <div class="card__code">Product code: 195434</div>
+          <div class="card__feedback">
+            <div class="card__stars">
+              <div class="card__star"></div>
+              <div class="card__star"></div>
+              <div class="card__star"></div>
+              <div class="card__star"></div>
+              <div class="card__star"></div>
+            </div>
+            <div class="card__review">Reviews: 5</div>
+          </div>
+          <div class="card__price">
+            <div class="card__price-name">Price:</div>
+            <div class="card__price-amount">$2,199</div>
+          </div>
+          <a
+            href="#"
+            class="card__button"
+          >
+            BUY
+          </a>
+        </div>
+      </div>
+
+      <div class="card">
+        <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="imac"
+        />
+        <div class="card__content">
+          <div class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </div>
+          <div class="card__code">Product code: 195434</div>
+          <div class="card__feedback">
+            <div class="card__stars">
+              <div class="card__star"></div>
+              <div class="card__star"></div>
+              <div class="card__star"></div>
+              <div class="card__star"></div>
+              <div class="card__star"></div>
+            </div>
+            <div class="card__review">Reviews: 5</div>
+          </div>
+          <div class="card__price">
+            <div class="card__price-name">Price:</div>
+            <div class="card__price-amount">$2,199</div>
+          </div>
+          <a
+            href="#"
+            class="card__button"
+          >
+            BUY
+          </a>
+        </div>
+      </div>
+
+      <div class="card">
+        <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="imac"
+        />
+        <div class="card__content">
+          <div class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </div>
+          <div class="card__code">Product code: 195434</div>
+          <div class="card__feedback">
+            <div class="card__stars">
+              <div class="card__star"></div>
+              <div class="card__star"></div>
+              <div class="card__star"></div>
+              <div class="card__star"></div>
+              <div class="card__star"></div>
+            </div>
+            <div class="card__review">Reviews: 5</div>
+          </div>
+          <div class="card__price">
+            <div class="card__price-name">Price:</div>
+            <div class="card__price-amount">$2,199</div>
+          </div>
+          <a
+            href="#"
+            class="card__button"
+          >
+            BUY
+          </a>
+        </div>
+      </div>
+    </main>
```

Comment:
The divs for each card should be wrapped in a container div. This will make it easier to style the cards as a grid and ensure they are properly aligned. Also, it's a good practice to use semantic HTML where possible. Consider using section tags to group related content.